lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -6,7 +6,7 @@
  */
  @use '../../scss-shared/nx-dark-mode-helpers';
 
- $primary-btn-focus-outline-spacing: 2px;
+ $-primary-btn-focus-outline-spacing: 2px;
```

Comment:
You can declare variables inside of scss blocks to scope them just to that block, which removes the need to make their names so detailed.

Reply:
> You can declare variables inside of scss blocks to scope them just to that block, which removes the need to make their names so detailed.

With the new css variable made (--nx-focus-outline-inset), this no longer seemed necessary so removed entirely here: [913d7f1](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/913d7f1ffccd46447a83a70c425b8bbd1cabc07e)